<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arac_ekle</class>
 <widget class="QWidget" name="arac_ekle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Araç Ekle</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Araç Bilgileri</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ARAÇ TÜRÜ:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="arac_tur_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sedan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pickup</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hatchback</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUV</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ARAÇ PLAKASI:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ABONELİK TÜRÜ:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="abone_tur_combobox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AYLIK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YILLIK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HAFTALIK</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="arac_plaka">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Abone Bilgileri</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SOYAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TELEFON NUMARASI:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ad_linedit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="soy_ad_linedit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tel_lineedit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="kaydet_arac">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_sil">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYIT SİL</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablo_arac">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>821</width>
     <height>401</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>AD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SOYAD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TEL</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ARAÇ TÜRÜ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ARAÇ TÜRÜ</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="kayit_listele">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYIT LİSTELE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plaka_oku_buton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAKA OKU</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_browse">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
